device-types/Cisco/N9K-C93360YC-FX2.yaml

```diff
@@ -0,0 +1,235 @@
+---
+manufacturer: Cisco
+model: Nexus 93360YC-FX2
+part_number: N9K-C93360YC-FX2
+slug: cisco-n9k-c93360yc-fx2
+u_height: 2
+is_full_depth: true
+console-ports:
+  - name: console
+    type: rj-45
+interfaces:
+  - name: Ethernet1/1
+    type: 10gbase-t
```

Comment:
The spec sheet for this states:

N9K-C93360YC-FX2 | Nexus 9K Fixed with 96p 1/10G/25G SFP and 12p 40G/100G QSFP28

For 25GB SFP you need to use `25gbase-x-sfp28` instead of `10gbase-t`